fastify.js

```diff
@@ -352,9 +352,11 @@ function fastify (options) {
   // - ready
   // - onClose
   // - close
+
+  const avvioPluginTimeout = Number(options.pluginTimeout)
   const avvio = Avvio(fastify, {
     autostart: false,
-    timeout: Number(options.pluginTimeout) || defaultInitOptions.pluginTimeout,
+    timeout: !isNaN(avvioPluginTimeout) ? avvioPluginTimeout : defaultInitOptions.pluginTimeout,
```

Comment:
```suggestion
    timeout: isNaN(avvioPluginTimeout) !== false ? avvioPluginTimeout : defaultInitOptions.pluginTimeout,
```

Reply:
`=== false` since this is a check if the variable is not NaN